Source/SwipeTableViewCell.swift

```diff
@@ -17,10 +17,9 @@ open class SwipeTableViewCell: UITableViewCell {
     /// The object that acts as the delegate of the `SwipeTableViewCell`.
     public weak var delegate: SwipeTableViewCellDelegate?
 
-    /// If the user swipes faster than this threshold, it will complete the primary action even if the user doesn't swipe far enough
-    /// Default value is nil which means this feature is disabled
-    /// Always enter a positive number. Behavior for negative numbers is undefined.
-    public var velocityThreshold: CGFloat? = nil
+    /// If the user swipes quickly or beyond a certain distance threshold, you may want to force the swipe through gesture
+    /// Every time a pan gesture finishes, this block is called. If this block returns true, it will not open the menu and instead trigger a swipe through animation
+    public var forceSwipeThroughGesture: ((UIPanGestureRecognizer)->Bool)? = nil
```

Comment:
customActionTrigger?

customPerformAction or shouldPerformAction for the other bool?